data/mods/DinoMod/obsolete/monsters.json

```diff
@@ -65,20 +65,20 @@
     "type": "MONSTER",
     "copy-from": "mon_compsognathus_hatchling",
     "name": "mon_obsoletefaction1",
-    "default_faction": "gallimimus"
```

Comment:
What is happening here?

Reply:
Gallimimus doesn't exist anymore as the name of a faction; it's now herbivore_dino - without changing this to a valid faction, you get errors on load.

Reply:
Okay to just delete the whole entry then? It was only there to prevent errors

Reply:
I will check to make sure that doesn't cause a problem (later this weekend); if no problems, will do.

Reply:
Done.